README.md

```diff
@@ -29,21 +29,23 @@ Use `lerna` for publishing a new version of all Defender Client packages (exclud
 
 The following publishes a release candidate with the npm tag `next`:
 
-```
+```bash
```

Comment:
it seems like we want to do all this publishing through Github CI now, right? If so, I feel like these instructions should be directing people to Github CI rather than publishing from their machine

Reply:
Yeah i think it's better to remove. I was thinking for folks to reference just in case.